airbyte-webapp/src/components/ui/NextTable/NextTable.tsx

```diff
@@ -26,7 +27,7 @@ export const NextTable = <TData,>({ columns, data, onClickRow, className }: Prop
             {headerGroup.headers.map((header) => (
               <th
                 colSpan={header.colSpan}
-                className={classNames(styles.th, header.column.columnDef.meta?.thClassName)}
+                className={classNames(styles.th, (header.column.columnDef.meta as ColumnMeta | undefined)?.thClassName)}
```

Comment:
:information_source: The type overwrite via `declare` doesn't work with `pnpm` anymore (I don't fully understand why, yet), so moving it into an in place cast instead.